### PR TITLE
Update generator.py

### PR DESCRIPTION
**Pull request by machinelearningdeveloper**

### CODE REVIEW

**Comment by s0md3v** [comment]
Hi there,

Thanks for contributing to the project.
Would you mind describing the changes?
It saves time for everyone.

Thanks.

**Comment by patzerr** [comment]
Hi,
looks like an attempt for clean(er) code by extracting stuff to functions, even the case for context == "script".

I can't  review how much the function names make sense.

hth

File: core/generator.py [hunk 1]
```diff
@@ -1,119 +1,125 @@
...
-                    for function in functions:
-                        if '=' in function:
-                            function = '(' + function + ')'
-                        if breaker == '':
-                            filling = ''
-                        vector = '\\' + breaker + closer + filling + function + suffix
-                        vectors[6].add(vector)
+            handle_comment_context(occurrence, breaker, special, vectors[10])
+        elif context == 'script' and scripts:
+            script = index < len(scripts) scripts[index] else scripts[0]
```
Comment (s0md3v) [inline core/generator.py]: [Invalid Syntax]

Comment (patzerr) [reply core/generator.py]: Not a python developer here, but looks like an inline if, or a trinity
```suggestion
            script = index < len(scripts) if scripts[index] else scripts[0]
```

**Comment by s0md3v** [comment]
Okay, I fixed that one and now new errors popped up.
Sorry but I have to close this one bud.

**Comment by patzerr** [comment]
Which you seem to have introduced ;)
htmlCcontext vs htmlContext

**Comment by s0md3v** [comment]
[closing due to inactivity, can be re-opened if author makes it work]

**Pull request closed without merging**